examples/nidcpower/measure-record.py

```diff
@@ -0,0 +1,196 @@
+# This example performs continuous measure record.
+# 
+# The gRPC API is built from the C API. NI-DCPower documentation is installed with the driver at:
+# C:\Program Files (x86)\IVI Foundation\IVI\Drivers\niDCPower\Documentation\NIDCPowerCref.chm
+#
+# Getting Started:
+#
+# To run this example, install "NI-DCPower Driver" on the server machine.
+# Link : https://www.ni.com/en-in/support/downloads/drivers/download.ni-dcpower.html
+#
+# For instructions on how to use protoc to generate gRPC client interfaces, see our "Creating a gRPC Client" wiki page.
+# Link: https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
+#
+# Refer to the NI DCPOWER gRPC Wiki for the latest C Function Reference:
+# Link: https://github.com/ni/grpc-device/wiki/NI-DCPOWER-C-Function-Reference
```

Comment:
[No Action] I assume this will be published when the feature is finished. Make sure to update this link if the planned wiki name changes between now and then.

Reply:
Definitely.